h/views/account_signup.py

```diff
@@ -163,6 +165,13 @@ class IDInfo:
     sub: str
     """The user's unique ID from the third-party provider."""
 
+    rfp: str
+    """A Request Forgery Protection (RFP) token to protect against CSRF attacks.
+
+    The claim name "rfp" is compatible with
+    https://datatracker.ietf.org/doc/html/draft-bradley-oauth-jwt-encoded-state-09
+    """
```

Comment:
Using `rfp` for the name of the CSRF token in the JWT, same as we do for OAuth 2 state JWTs.